pipeline.py

```diff
@@ -59,30 +82,74 @@ class TextProcessingPipeline:
     Process articles from corpus manager
     """
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
+        self.raw_text = ''
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles()
+        for article in articles.values():
+            self.raw_text = article.get_raw_text()
+            list_of_tokens = self._process()
+            list_of_strs = []
+            for token in list_of_tokens:
+                list_of_strs.append(token.__str__())
+            article.save_processed(' '.join(list_of_strs))
 
     def _process(self) -> List[type(MorphologicalToken)]:
         """
         Performs processing of each text
         """
+        result = Mystem().analyze(self.raw_text)
+        morph_analyzer = MorphAnalyzer()
+        list_of_tokens = []
+        for stem_dict in result:
+            if stem_dict.get('analysis'):
+                original_word = stem_dict['text']
```

Comment:
you have checked for the single key - analysis. however, there are more keys to check: lex, gr